specification/peering/resource-manager/Microsoft.Peering/preview/2019-03-01-preview/peering.json

```diff
@@ -0,0 +1,1309 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-03-01-preview",
+    "title": "PeeringManagementClient",
+    "description": "APIs to manage Peering resources through the Azure Resource Manager."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Peering/legacyPeerings": {
+      "get": {
+        "tags": [
+          "LegacyPeerings"
+        ],
+        "description": "Lists all of the legacy peerings under the given subscription matching the specified kind and location.",
+        "operationId": "LegacyPeerings_List",
+        "parameters": [
+          {
+            "name": "peeringLocation",
```

Comment:
>peeringLocation [](start = 21, length = 15)

Is this the peering location itself (ex: Redmond) or peeringLocation resource name ? (peeringLocationFoo that represents Redmond,WA). 
I would consider taking peering location resource uri, as part of request body. For ex: names can clash.. just peeringLocation=Redmond is ambiguous it can be present in multiple states/countries)
Does that make sense ?